fastify.js

```diff
@@ -59,6 +59,28 @@ const onBadUrlContext = {
   onError: []
 }
 
+const caseInsensitiveParse = (string) => {
```

Comment:
Do you guys think we can use `qs` when `caseSensitive` is `false`? Much simpler than the custom function.

```js
qs.parse(str, {
  decoder: (str, encoder, _, type) => type === 'key' ? encoder(str.toLowerCase()) : encoder(str)
})
```

_Although I agree with [this](https://github.com/fastify/fastify/issues/2644#issuecomment-715498064), simply using custom `querystringParser` is the recommended fix in this case._

Reply:
I wouldn't add dependencies but in general I'm ok

Reply:
I don't want to add qs to the deps. We might also just document if that somebody wants case insensitive query string keys.. then they can use qs. This is already possible, and maybe this PR is actually not needed at all.

Reply:
If that's the case, I can add that and close this one, just give me a confirmation

Reply:
Yeah, docs saying "if you need this, use `qs`" is the easier and better solution.

Reply:
Let's go for that.